### PR TITLE
Fixed jsp compilation errors - alpaca branch

### PR DESCRIPTION
**Pull request by kateyang1998**
- Added jspc dependency to compile the jsp files
- Fixed jsp compilation errors
- Excluded all jspf files for now, excluded 1 deprecated jsp file and 2 jsp files containing HTML taglib errors to avoid error detections when compiling. (Will look into these files again)

## Summary by Sourcery

Bug Fixes:
- Fix JSP compilation errors by adding necessary dependencies and excluding problematic files.

### CODE REVIEW

File: src/main/webapp/PMmodule/GenericIntake/Edit.jsp [hunk 1]
```diff
@@ -743,4 +742,4 @@
 </div>
 </html:form>
```
Comment (yingbull) [inline src/main/webapp/PMmodule/GenericIntake/Edit.jsp:743]: Html form was originally there?

File: src/main/webapp/PMmodule/GenericIntake/Print.jsp [hunk 2]
```diff
@@ -22,94 +22,94 @@
     Toronto, Ontario, Canada
 
 --%>
-<%@ include file="/taglibs.jsp"%>
-<%@ page import="org.oscarehr.PMmodule.model.Intake"%>
+<%@ include file="/taglibs.jsp" %>
+<%@ page import="org.oscarehr.PMmodule.model.Intake" %>
 <%@ page
-	import="org.oscarehr.PMmodule.web.formbean.GenericIntakeEditFormBean"%>
+        import="org.oscarehr.PMmodule.web.formbean.GenericIntakeEditFormBean" %>
```
Comment (yingbull) [inline src/main/webapp/PMmodule/GenericIntake/Print.jsp:28]: Fix indent

File: src/main/webapp/PMmodule/GenericIntake/PrintPreview.jsp [hunk 3]
```diff
@@ -22,98 +22,98 @@
     Toronto, Ontario, Canada
 
 --%>
-<%@ include file="/taglibs.jsp"%>
-<%@ page import="org.oscarehr.PMmodule.model.Intake"%>
+<%@ include file="/taglibs.jsp" %>
+<%@ page import="org.oscarehr.PMmodule.model.Intake" %>
 <%@ page
-	import="org.oscarehr.PMmodule.web.formbean.GenericIntakeEditFormBean"%>
+        import="org.oscarehr.PMmodule.web.formbean.GenericIntakeEditFormBean" %>
```
Comment (yingbull) [inline src/main/webapp/PMmodule/GenericIntake/PrintPreview.jsp:28]: Fix indent

File: src/main/webapp/PMmodule/IntakeCReport.jsp [hunk 4]
```diff
@@ -23,81 +23,85 @@
...
-			%>
-</table>
-</body>
-</html:html>
+<html>
+    <head>
+        <script type="text/javascript" src="<%= request.getContextPath() %>/js/global.js"></script>
+        <title>Street Health Mental Health Report</title>
+        <link rel="stylesheet" type="text/css"
+              href="<html:rewrite page="/css/intakeC.css" />"/>
```
[the remark this answers is not in the document]
Comment (yingbull) [reply src/main/webapp/PMmodule/IntakeCReport.jsp:46]: I still see html rewrite not jstl and EL

**Review by yingbull — CHANGES_REQUESTED** [review]

File: src/main/webapp/PMmodule/GenericIntake/Edit.jsp (hunk 1 above)
Comment (kateyang1998) [reply src/main/webapp/PMmodule/GenericIntake/Edit.jsp:743]: Yes, it is the original version.

File: src/main/webapp/PMmodule/GenericIntake/Print.jsp (hunk 2 above)
Comment (kateyang1998) [reply src/main/webapp/PMmodule/GenericIntake/Print.jsp]: Done

**Pull request merged**